hubblestack/fileclient.py

```diff
@@ -813,23 +775,15 @@ def get_file(self,
         if not dest2check:
             rel_path = self._check_proto(path)
 
-            log.debug(
-                'In saltenv \'%s\', looking at rel_path \'%s\' to resolve '
-                '\'%s\'', saltenv, rel_path, path
-            )
-            with self._cache_loc(
-                    rel_path, saltenv, cachedir=cachedir) as cache_dest:
+            log.debug("In saltenv '%s', looking at rel_path '%s' to resolve " "'%s'", saltenv, rel_path, path)
```

Comment:
Unecessary " " in middle og log.debug.   Maybe change to:
`log.debug("In saltenv '%s', looking at rel_path '%s' to resolve '%s'", saltenv, rel_path, path)